### PR TITLE
pulls chrom. lengths from the same source as for the bigwig files.

### PR DESCRIPTION
**Pull request by Blosberg**
Address issue #73 --tested, and produces report successfully. Still need to confirm whether this is the only instance of unwanted internet-fetching (but this does certainly eliminate at least one).

### CODE REVIEW

File: BSseq_pipeline.py [hunk 1]
```diff
@@ -566,12 +566,14 @@
         rules.bam_methCall.output,
         rules.methseg.output,
         lambda wc: finalReportDiffMeth_input(wc.prefix),
-        template      = os.path.join(DIR_templates,"index.Rmd")
+        template      = os.path.join(DIR_templates,"index.Rmd"),
+        chrom_seqlengths    = DIR_mapped+"Refgen_"+ASSEMBLY+"_chromlengths.csv"
```
Comment (rekado) [inline BSseq_pipeline.py]: I'd prefer using os.path.join here.

File: BSseq_pipeline.py [hunk 2]
```diff
@@ -566,12 +566,14 @@
...
     output: 
         report        = os.path.join(DIR_final, "{prefix}.sorted_{assembly}_final.html")
     params:
         ## absolute path to bamfiles
         Samplename  = lambda wc: get_fastq_name( wc.prefix ),
+        chrom_seqlengths  = DIR_mapped+"Refgen_"+ASSEMBLY+"_chromlengths.csv",
```
Comment (rekado) [inline BSseq_pipeline.py]: and here.

File: report_templates/index.Rmd.in [hunk 3]
```diff
@@ -734,14 +735,12 @@ cat('### Overview of Hyper- and Hypo-Methylated CpGs Over the Genome\n')
 
 if(length(GRanges.diffmeth.hypo)>1 & length(GRanges.diffmeth.hyper)>1){
 
-require(GenomeInfoDb)
```
Comment (rekado) [inline report_templates/index.Rmd.in]: Can we drop GenomeInfoDb from the dependencies then?

Comment (Blosberg) [reply report_templates/index.Rmd.in]: I'll remove ` library("GenomeInfoDb")` from the index.Rmd file on the next commit and re-test

**Pull request merged**